lib-ce/emqx_dashboard/src/emqx_dashboard_admin.erl

```diff
@@ -207,16 +270,20 @@ is_valid_pwd(<<Salt:4/binary, Hash/binary>>, Password) ->
 %%--------------------------------------------------------------------
 
 init([]) ->
-    case binenv(default_user_username) of
-        <<>> -> ok;
-        UserName ->
-            %% Add default admin user
-            {ok, _} = mnesia:subscribe({table, mqtt_admin, simple}),
-            PasswordHash = ensure_default_user_in_db(UserName),
-            ok = ensure_default_user_passwd_hashed_in_pt(PasswordHash),
-            ok = maybe_warn_default_pwd()
-    end,
```

Comment:
move this part to a sub function:
```
boostrap_default_admin_user() ->
    ...
```

and `init([])` can be kept:
```
case add_bootstrap_users() of
    ok -> boostrap_default_admin_user();
    Error -> {stop, Error}
end.
```

Reply:
done